examples/simpleapp/readme.md

```diff
@@ -35,41 +35,26 @@ Why it is "isolated"? Because inside SC genesis block declared a MC Block Refere
 Such a configuration is useful when you want just to test Sidechain logic and behaviour without any interaction with a Mainchain network.
 
 
-**Running SimpleApp connected to the real MC node**
-
-To run SC with a connection with a real MC node do the following steps:
 
-1. Compile MC sources with SC support code:
-    * clone Horizen core repository - https://github.com/ZencashOfficial/zen
-    * switch to the SC suport branch - `as/sc_development`
-    * build the Core for your platform using the guides in the repo.
-2. Run MC node in a `regtest` mode with a `-websocket` flag. 
-3. Generate at least 220 blocks to enable SC support.
-4. Declare the new SC using `sc_create` RPC method, where you will specify new SC id and first forward transfers.
-5. Generate 1 more block, that includes the transaction previously generated.
-6. Get SC genesis info from MC for your SC ID using `getscgenesisinfo` RPC method.
-7. Run ScBootstrapping tool and execute a command `genesisinfo`, where the result from previous RPC method should be placed as well as a secret, that will be used to sign the SC genesis block.
-8. Use the result data and update configuration file with it. You can use configuration files from `./examples/simpleapp/src/main/resourse/` as an example of resulting structure of the file.
-9. Run SimpleApp node with the configuration file you created. Your SC node will be connected to the MC node.
-10. If new MC blocks will appear, than the next SC blocks will automatically include the references to MC blocks inside. Maximum number of MC Block Refs per SC Block is 3.
+**Running SimpleApp connected to the real MC node**
```

Comment:
I would prefer to leave a short version here too and make it actual.. Moreover here we specify the proper MC git branch and some additional data that is not present in `mc_sc_workflow_example.md`